api/v1alpha/framework/xcuitest.schema.json

```diff
@@ -84,6 +88,10 @@
             "description": "Description for the testApp",
             "type": "string"
           },
+          "otherApps": {
+            "description": "A list of applications to be installed alongside the main app. Applications can be defined as a local path or a remote url. If a remote url is defined, the app will be downloaded to a local temp directory before uploading to the SauceLabs Mobile App Storage service. Supports environment variables as values.",
+            "type": "array"
+          },
```

Comment:
We should also note here that only a max. number of 2 apps in otherApps is supported when using simulartors. RDC is fine.

Reply:
👍 added